vita3k/gui/src/content_manager.cpp

```diff
@@ -185,43 +185,43 @@ static bool get_size_selected_contents(GuiState &gui, HostState &host) {
     return contents_size;
 }
 
-struct Dlc {
+struct addcont {
```

Comment:
fail lol, struct is all time first letter in shift

Reply:
i got it.😭

Reply:
so here is AddCont